user_documentation_cf_cli_plugin.md

```diff
@@ -179,6 +185,66 @@ Restore has been aborted for the instance name: [SERVICE\_INSTANCE\_NAME]
 
 **Additional note:** The successful execution of this command means the abort process was initiated. Theprocess of aborting the backup again takes some time to complete. For the convenience of the user, the abort process too runs in the background. If you wish to know the progress and/or the state of the backup, you can use the &quot;cf service SERVICE\_INSTANCE\_NAME&quot; command.
 
+### Listing backups for deleted service instances:
```

Comment:
Just to keep inline with above,
"Listing all backups of a deleted service-instance"